templates/bot/csharp/workflow/GettingStarted.txt

```diff
@@ -0,0 +1,27 @@
+Welcome to Teams Toolkit!
+
+Quick Start
+-------------------------
+1. Download and install ngrok (https://ngrok.com/)
+2. Open a command prompt and run ngrok http 5130
+3. Right-click your project and select Teams Toolkit > Prepare Teams app dependencies
+4. If prompted, sign in with an M365 account for the Teams organization you want 
+to install the app to
+5. Press F5, or select the Debug > Start Debugging menu in Visual Studio
+6. In the launched browser, select the Add button to load the app in Teams
+7. In the chat bar, type and send "helloWorld" to your app to trigger a response
+
```

Comment:
Better to add tutorial for "doStuff" as well.

Reply:
Let me add the aka link to the action handler tutorial here.